CAIPs/caip-77.md

```diff
@@ -0,0 +1,69 @@
+---
+caip: 77
+title: Asset Reference for the Hedera token namespace
+author: Danno Ferrin (@shemnon)
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/169
+status: Draft
+type: Standard
+created: 2021-11-01
+updated: 2021-11-01
+requires: 19, 75
+---
+
+## Simple Summary
+
+This document is about the details of the Hedera Token Service 'token' asset
+namespace and reference for CAIP-19.
+
+## Abstract
+
+In CAIP-19 a general asset identification scheme is defined. This is the
+implementation of CAIP-19 for `token` in the
+[Hedera Token Service](https://docs.hedera.com/guides/docs/integrating-a-hedera-token-service-token#hedera-token-service)
+representing fungible tokens.
+
+## Motivation
+
+See CAIP-19.
+
+## Specification
+
+### Token Asset Namespace
+
+The asset namespace is called "token" hosted in the Hedera Token Service (HTS).
+It references HTS tokens in the `hedera` namespace (see CAIP-xx).
```

Comment:
please replace xx with the number - should be 76 right?

Reply:
#75 actually, still needs a review.

Reply:
done